ccp/apis/v1/ccp_server/backend/builds/build_logs.py

```diff
@@ -102,7 +105,7 @@ def response(namespace, appid, jobid, desired_tag, build):
                 namespace,
                 jenkins_job_name
             ],
-            build_number=build
+            build_number=build_number
```

Comment:
Same comment as earlier for `build_number`.

Reply:
its not kind of variable assignment